hips/utils/healpix.py

```diff
@@ -54,3 +56,33 @@ def boundaries(nside: int, pix: int, nest: bool=True) -> tuple:
     boundary_coords = hp.boundaries(nside, pix, nest=nest)
     theta, phi = hp.vec2ang(np.transpose(boundary_coords))
     return theta, phi
+
+def compute_image_pixels(nside: int, shape: tuple, wcs: WCS) -> np.ndarray:
+    """Returns an array containing the pixels corresponding to an image.
+
+    This function calls `healpy.pixelfunc.ang2vec`, `healpy.query_disc`, and
+    `astropy.coordinates.angle_utilities.angular_separation` to compute
+    the pixel values, which will be use in tile drawing.
+
+    Parameters
+    ----------
+    nside : int
+        The nside of the HEALPix map
+    shape : tuple
+        Shape of the image
+    wcs : astropy.wcs.wcs.WCS
+        A WCS object containing the image header
+
+    Returns
+    -------
+    pixels : `numpy.ndarray`
+        Returns a list of pixel values
```

Comment:
This description of the return value is not accurate. It's not "pixel values", but "HEALPix pixel numbers" that are computed and returned here. Pixel values would be the something like "counts" or "brightness" or whatever quantity is stored in the pixels.